src/client.rs

```diff
@@ -263,7 +405,27 @@ impl UdpCoAPClient {
     }
 
     pub fn set_broadcast(&self, value: bool) -> IoResult<()> {
-        self.transport.inner.socket.set_broadcast(value)
+        self.transport.transport.socket.set_broadcast(value)
+    }
+
+    /// creates a receiver based on a specific request
+    /// this method can be used if you send a multicast request and
+    /// expect multiple responses.
+    /// only use this method if you know what you are doing
+    pub async fn create_receiver_for(
```

Comment:
I think it would be better if you can provide an example how to use the function to handle multiple responses

Reply:
added example!